src/site/content/en/react/how-to-use-amp-in-nextjs-and-why/index.md

```diff
@@ -0,0 +1,272 @@
+
+   const imgSrc = 'https://placekitten.com/1000/1000';
+
+   const Image = () => (
+     useAmp() ? (
+       <amp-img alt="A cute kitten"
+         src={imgSrc}
+         width="1000"
+         height="1000"
+         layout="responsive">
+       </amp-img>
+     ) : (
+       <img alt="A cute kitten"
+         src={imgSrc}
+         width="1000"
+         height="1000">
+       </img>
+     )
+   );
+
+   const Home = () => (
+     <div>
+       <Image />
+     </div>
+   );
+
+   export default Home;
+   ```
+
+   `layout="responsive"` automatically renders a fully responsive image with an aspect ratio
+   specified by width and height. Check out [Layout & media queries][layout] to learn more about
+   the supported layouts of AMP elements, and [amp-img] to learn more about that element's
+   optimizations.
+    
+1. View the main version of the page again.
+
+   <figure>
+     <img src="mainimg.png"
+          alt="The main version of the image.">
+     <figcaption>
+       The main version of the image.
+     </figcaption>
+   </figure>
+
+1. View the AMP version of the page again.
+
+   <figure>
+     <img src="ampimg.png"
+          alt="The AMP version of the image.">
+     <figcaption>
+       The AMP version of the image.
+     </figcaption>
+   </figure>
+
+### How to create AMP-only pages {: #amponly }
+
+Next.js also supports AMP-only pages. With this approach, a single AMP page is served and rendered
+to users and search engines at all times.
+
+1. To render an AMP-only page, change the value of the `amp` property in the config object to `true`.
+
+   ```jsx
+   import React from 'react'
+
+   export const config = { amp: true };
+
+   const Home = () => (
+     <p>This is an AMP-only page</p>
+   );
+     
+   export default Home;
+   ```
+
+## How to decide whether to use the Hybrid AMP or AMP-only approach {: #guidance }
+
+If you're serious about load performance, an AMP-only page could be a good way to make sure
+that your page gets fast and stays fast. But here's the catch: in order to guarantee fastness,
+AMP must prohibit certain architectural patterns and HTML elements that often lead to slow pages.
+For example, AMP doesn't allow custom synchronous JavaScript because 
+[render-blocking resources][blockers] are a common cause of slow page loads.
+
+In order to understand whether an AMP-only approach is right for you, you need to figure out
+whether all of your frontend code can be represented in AMP HTML:
+
+* Read [How AMP works](https://amp.dev/about/how-amp-works/) to understand AMP's high-level
+  architectural restrictions and mandatory optimizations.
+* Read [HTML Tags][tags] to see what HTML tags AMP allows and prohibits, and
+  browse the [AMP component catalogue](https://amp.dev/documentation/components/) to see the
+  custom components that the AMP community has built to solve common use cases.
+
+Even if an AMP-only approach won't work for your page1000, it might still be a good idea to
```

Comment:
Most likely a deliberate marker that you placed, but just in case you didn't:

```suggestion
Even if an AMP-only approach won't work for your page, it might still be a good idea to
```